woocommerce/woocommerce
# update Github with the latest build

### Changes proposed in this Pull Request:

In #43575 I updated the `md-docs` util but did not include an updated build of `monorepo-utils`. This PR updates the build from the latest `trunk`.

#43534 updates the docs manifest to `trunk` so this PR does not include a manifest update to avoid conflicts/ 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Run `pnpm utils md-docs create docs woocommerce`
2. Run `diff manifest.json docs/docs-manifest.json | grep tags:` as adding tags support was added in #43575
3. The diff should not produce output

**Note:** the CI matrix task will test the new build as well



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
